contracts/governance/IGovernor.sol

```diff
@@ -232,6 +235,10 @@ abstract contract IGovernor is IERC165, IERC6372 {
      *
      * NOTE: The {votingDelay} can delay the start of the vote. This must be considered when setting the voting
      * duration compared to the voting delay.
+     *
+     * NOTE: This value is saved when the proposal is saved, so that possible changes to the value do not affect
```

Comment:
`saved` is kinda ambiguous, I think it might be better to:

```suggestion
     * NOTE: This value is stored when the proposal is submitted so that possible changes to the value do not affect
```